cachers/combined.go

```diff
@@ -94,17 +79,18 @@ func (l *CombinedCache) Get(ctx context.Context, actionID string) (outputID, dis
 
 func (l *CombinedCache) Put(ctx context.Context, actionID, outputID string, size int64, body io.Reader) (string, error) {
 	pr, pw := io.Pipe()
-	diskPutCh := make(chan any, 1)
+	diskpathCh := make(chan string, 1)
+	errorsCh := make(chan error, 1)
```

Comment:
```suggestion
	diskPathCh := make(chan string, 1)
	errCh := make(chan error, 1)
```

`errCh` is a convention